apps/src/music/views/MusicView.jsx

```diff
@@ -643,36 +646,43 @@ class UnconnectedMusicView extends React.Component {
   };
 
   render() {
+    console.log(this.props);
     return (
       <AnalyticsContext.Provider
         value={this.props.onProjectBeats ? this.analyticsReporter : null}
       >
-        <KeyHandler
-          togglePlaying={this.togglePlaying}
-          playTrigger={this.playTrigger}
-          uiShortcutsEnabled={
-            AppConfig.getValue('ui-keyboard-shortcuts-enabled') === 'true'
-          }
-        />
-        <MusicLabView
-          blocklyDivId={BLOCKLY_DIV_ID}
-          setPlaying={this.setPlaying}
-          playTrigger={this.playTrigger}
-          hasTrigger={id => this.musicBlocklyWorkspace.hasTrigger(id)}
-          getCurrentPlayheadPosition={this.getCurrentPlayheadPosition}
-          updateHighlightedBlocks={this.updateHighlightedBlocks}
-          undo={this.undo}
-          redo={this.redo}
-          clearCode={this.clearCode}
-          validator={this.musicValidator}
-          player={this.player}
-          allowPackSelection={
-            this.library?.getHasRestrictedPacks() &&
-            !this.props.levelProperties?.levelData?.packId &&
-            this.props.isProjectLevel
-          }
-        />
-        <Callouts />
+        {/*<KeyHandler*/}
+        {/*  togglePlaying={this.togglePlaying}*/}
+        {/*  playTrigger={this.playTrigger}*/}
+        {/*  uiShortcutsEnabled={*/}
+        {/*    AppConfig.getValue('ui-keyboard-shortcuts-enabled') === 'true'*/}
+        {/*  }*/}
+        {/*/>*/}
+        <MusicPlayView />
+        {/*<MiniPlayerView*/}
+        {/*  projects={[{name: 'test', id: 1}]}*/}
+        {/*  libraryName={this.props.libraryName}*/}
+        {/*/>*/}
+
+        {/*<MusicLabView*/}
+        {/*    blocklyDivId={BLOCKLY_DIV_ID}*/}
+        {/*    setPlaying={this.setPlaying}*/}
+        {/*    playTrigger={this.playTrigger}*/}
+        {/*    hasTrigger={id => this.musicBlocklyWorkspace.hasTrigger(id)}*/}
+        {/*    getCurrentPlayheadPosition={this.getCurrentPlayheadPosition}*/}
+        {/*    updateHighlightedBlocks={this.updateHighlightedBlocks}*/}
+        {/*    undo={this.undo}*/}
+        {/*    redo={this.redo}*/}
+        {/*    clearCode={this.clearCode}*/}
+        {/*    validator={this.musicValidator}*/}
+        {/*    player={this.player}*/}
+        {/*    allowPackSelection={*/}
+        {/*        this.library?.getHasRestrictedPacks() &&*/}
+        {/*        !this.props.levelProperties?.levelData?.packId &&*/}
+        {/*        this.props.isProjectLevel*/}
+        {/*    }*/}
+        {/*/>*/}
+        {/*<Callouts />*/}
```

Comment:
Just as a suggestion - for commenting out big blocks like this you can also just wrap the entire set of blocks in `{/* ... */}`